TeamSnapSDK/SDK/DataTypes/Messages/TSDKMessageDatum.h

```diff
@@ -16,6 +22,9 @@
 @property (nonatomic, weak) NSURL *_Nullable linkUser;
 @property (nonatomic, weak) NSURL *_Nullable linkMember;
 
+- (TSDKMessageDatumMessageType)messageTypeOfUnreadCount;
++ (NSString *_Nullable)stringValueForMessageType:(TSDKMessageDatumMessageType)messageType;
```

Comment:
I would normally suggest adding nullability headers for touched obj-c headers, but it looks like every property on this file is already nullable.